docs/topics/runbooks/dev/shell_into_dev_pod.md

```diff
@@ -0,0 +1,59 @@
+# Shell into a dev pod
+
+## Steps
+
+### Shell into bastion host
+
+This will authenticate you for gcloud and open a shell into the bastion host.
+
+```bash
+gcloud auth login --update-adc
+gcloud compute ssh --zone "us-west1-a" "bastion-us-west1" --project "moz-fx-bastion-nonprod-global" --tunnel-through-iap
```

Comment:
restarting our conversation from the other PR - I don't think we should duplicate these instructions with SRE's instructions for shell'ing into the dev pod on mana. My feeling is that it's best left in SRE's maintaining hands, because they control the infrastructure, and would be best to keep the urls, instructions, etc, up to date - but I'd sure they wouldn't mind us improving it.  So just link to it here?

Reply:
Think you're right. Instead of having this dock, we can simply link to the confluence page. This particular section is not really relevant for non-mozilla employees anyway.

Reply:
I agree, I think we should leave this on Confluence SRE page